roles/sssd/tasks/main.yml

```diff
@@ -148,6 +148,40 @@
     - ansible_facts['os_family'] == "RedHat"
     - ansible_facts['distribution_major_version'] <= "7"
 
+# LDAP backups
+- name: Create /root/ldap_backups
+  ansible.builtin.file:
+    path: /root/ldap_backup
+    state: directory
+    owner: root
+    group: root
+    mode: '0700'
+  become: true
+
+- name: 'Deploy ldap_backup script'
+  ansible.builtin.template:
+    src: ldap_backup.sh.j2
+    dest: /root/ldap_backup.sh
+    owner: root
+    group: root
+    mode: '0700'
+  when:
+    - ldap_domains is defined
+    - ldap_domains | length > 0
+    - create_ldap is false
+    - inventory_hostname in groups['sys_admin_interface'] | default([])
+  become: true
+
+- name: 'Install daily cron for ldap backups'
+  ansible.builtin.cron:
+    name: 'Make daily ldap backup - do not change it manually!'
+    hour: '3'
+    minute: '57'
+    user: 'root'
+    disabled: false
+    job: /root/ldap_backup.sh
+  become: true
+
```

Comment:
Three tasks are added, but only one has
```
  when:
    - ldap_domains is defined
    - ldap_domains | length > 0
    - create_ldap is false
    - inventory_hostname in groups['sys_admin_interface'] | default([])
```
This means the folder and cron job are created on all machines, but the backup script is only created on the SAIs. On all other machines the cronjob will always fail... 
I suggest to move all three tasks without when into a separate ```ldap_backup.yml``` and include that like this
```
- name: Create LDAP backups.
  ansible.builtin.include_tasks:
    file: ldap_backup.yml
  when:
    - ldap_domains is defined
    - ldap_domains | length > 0
    - create_ldap is false
    - inventory_hostname in groups['sys_admin_interface'] | default([])
```

Reply:
Uf. You are right. Will move into own .yml indeed.